proxyroles/expressionrole.h

```diff
@@ -23,15 +23,17 @@ class ExpressionRole : public SingleRole
 
 Q_SIGNALS:
     void expressionChanged();
-
+    
 private:
-    QVariant data(const QModelIndex& sourceIndex, const QQmlSortFilterProxyModel& proxyModel) override;
-    void updateContext(const QQmlSortFilterProxyModel& proxyModel);
+    QVariant extracted();
```

Comment:
where is this coming from? Can't find the implementation

Reply:
Reverted, thx for pointing out